conf/ocp_version/ocp-4.2-ga-config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+# Config file for OCP GA 4.2 stable channel
+
+RUN:
+  client_version: '4.2-ga'
+
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.2-ga"
+  ocp_channel: "stable"
+  #ocp_version_minus: 0
```

Comment:
Not sure, what this `ocp_version_minus` means? I didn't find any reference anywhere else.